components/collection/unlockable/UnlockableCollectionBanner.vue

```diff
@@ -2,9 +2,15 @@
   <div class="collection-banner" :style="{ backgroundImage: `url(${banner})` }">
     <div class="collection-banner-shadow"></div>
 
-    <section class="h-full py-8">
-      <div class="container is-fluid collection-banner-content">
-        <div class="flex flex-col items-start">
+    <section class="h-[368px] lg:h-full">
+      <div
+        :class="[
+          'container h-full flex',
+          'justify-end items-start flex-col',
+          'md:justify-between md:items-end md:flex-row',
```

Comment:
multiple line?

Reply:
Original purpose is to organize and easily identify classes when dealing with many class names. Updated.